### PR TITLE
V1.1.0 minoradditions to add Provenance and Device

### PR DESCRIPTION
**Pull request by jadedavies11**
Add new profiles for Provenance and Device including valuesets and extensions

### CODE REVIEW

File: Profiles/Profile-DataStandardsWales-Patient.xml
Comment (callumsaint) [inline Profiles/Profile-DataStandardsWales-Patient.xml:1]: Change version to 1.1
change the death notification status minimum cardinality back to 0

Comment (jadedavies11) [reply Profiles/Profile-DataStandardsWales-Patient.xml:1]: updated

File: ValueSets/ValueSet-DataStandardsWales-ProvenanceActivity.xml
Comment (callumsaint) [inline ValueSets/ValueSet-DataStandardsWales-ProvenanceActivity.xml:1]: Change version from 0.0.5 to 0.0.1

**Review by callumsaint — CHANGES_REQUESTED** [review]
Changes requested in comments

File: ValueSets/ValueSet-DataStandardsWales-ProvenanceActivity.xml
Comment (jadedavies11) [reply ValueSets/ValueSet-DataStandardsWales-ProvenanceActivity.xml:1]: updated

File: Extensions/Extension-DataStandardsWales-CDR-PatientRecordType.xml
Comment (callumsaint) [inline Extensions/Extension-DataStandardsWales-CDR-PatientRecordType.xml:1]: Add date

File: Extensions/Extension-DataStandardsWales-CDR-SourceTimestamp.xml
Comment (callumsaint) [inline Extensions/Extension-DataStandardsWales-CDR-SourceTimestamp.xml:1]: Add date

File: Extensions/Extension-DataStandardsWales-DemographicsAsRecorded.xml
Comment (callumsaint) [inline Extensions/Extension-DataStandardsWales-DemographicsAsRecorded.xml:1]: Add date

File: NamingSystems/NamingSystem-ABUHB-pas-identifier.xml
Comment (callumsaint) [inline NamingSystems/NamingSystem-ABUHB-pas-identifier.xml:1]: Update date

File: NamingSystems/NamingSystem-BCU-Central-pas-identifier.xml
Comment (callumsaint) [inline NamingSystems/NamingSystem-BCU-Central-pas-identifier.xml:1]: Update date

File: NamingSystems/NamingSystem-BCU-East-pas-identifier.xml
Comment (callumsaint) [inline NamingSystems/NamingSystem-BCU-East-pas-identifier.xml:1]: Update date

File: NamingSystems/NamingSystem-BCU-West-pas-identifier.xml
Comment (callumsaint) [inline NamingSystems/NamingSystem-BCU-West-pas-identifier.xml:1]: Update date

File: NamingSystems/NamingSystem-CAVUHB-pas-identifier.xml
Comment (callumsaint) [inline NamingSystems/NamingSystem-CAVUHB-pas-identifier.xml:1]: Update date

File: NamingSystems/NamingSystem-HDUHB-pas-identifier.xml
Comment (callumsaint) [inline NamingSystems/NamingSystem-HDUHB-pas-identifier.xml:1]: Update date

File: NamingSystems/NamingSystem-PTHB-pas-identifier.xml
Comment (callumsaint) [inline NamingSystems/NamingSystem-PTHB-pas-identifier.xml:1]: Update date

File: NamingSystems/NamingSystem-SBUHB-pas-identifier.xml
Comment (callumsaint) [inline NamingSystems/NamingSystem-SBUHB-pas-identifier.xml:1]: Update date

File: NamingSystems/NamingSystem-VUNHST-canisc-identifier.xml
Comment (callumsaint) [inline NamingSystems/NamingSystem-VUNHST-canisc-identifier.xml:1]: Update date

File: NamingSystems/NamingSystem-VUNHST-pas-identifier.xml
Comment (callumsaint) [inline NamingSystems/NamingSystem-VUNHST-pas-identifier.xml:1]: Update date

File: Extensions/Extension-DataStandardsWales-CDR-PatientRecordType.xml
Comment (callumsaint) [reply Extensions/Extension-DataStandardsWales-CDR-PatientRecordType.xml:1]: Create a new valueset for gold and silver
Add it as a binding to the extension

**Review by callumsaint — CHANGES_REQUESTED** [review]
changes in comments

File: CodeSystems/CodeSystem-DataStandardsWales-PatientRecordType.xml
Comment (callumsaint) [inline CodeSystems/CodeSystem-DataStandardsWales-PatientRecordType.xml:1]: Is it WRRS or CDR?

Comment (callumsaint) [reply CodeSystems/CodeSystem-DataStandardsWales-PatientRecordType.xml:1]: Should we add CDR as a prefix to the resource names?

File: ValueSets/ValueSet-DataStandardsWales-PatientRecordType.xml
Comment (callumsaint) [inline ValueSets/ValueSet-DataStandardsWales-PatientRecordType.xml:1]: Double checking if we should add CDR to the name

File: Profiles/Profile-DataStandardsWales-Patient.xml
Comment (callumsaint) [inline Profiles/Profile-DataStandardsWales-Patient.xml:1]: The canonical URLs for the extensions need to be updated to what we have renamed them to

Comment (callumsaint) [reply Profiles/Profile-DataStandardsWales-Patient.xml:1]: I believe the slice name also has to be renamed to a lowercase cdr.

File: Extensions/Extension-DataStandardsWales-CDR-PatientRecordType.xml
Comment (callumsaint) [inline Extensions/Extension-DataStandardsWales-CDR-PatientRecordType.xml:1]: Should the record type cardinality be 1..1?

**Review by callumsaint — CHANGES_REQUESTED** [review]
changes in comments

File: CodeSystems/CodeSystem-DataStandardsWales-PatientRecordType.xml
Comment (jadedavies11) [reply CodeSystems/CodeSystem-DataStandardsWales-PatientRecordType.xml:1]: I've just removed WRRS and left the resources generic for now - we could always change them

File: Extensions/Extension-DataStandardsWales-CDR-PatientRecordType.xml
Comment (jadedavies11) [reply Extensions/Extension-DataStandardsWales-CDR-PatientRecordType.xml:1]: Yeah it makes the most sense to do that

**Review by callumsaint — APPROVED** [review]

**Pull request merged**